lib/secure_headers/headers/content_security_policy.rb

```diff
@@ -212,11 +212,7 @@ def populate_nonces(directive, source_list)
     # unsafe-inline, this is more concise.
     def append_nonce(source_list, nonce)
```

Comment:
You may want to remove the "depending on browser" part from the method comments.